zubhub_frontend/zubhub/src/views/edit_profile/editProfileScripts.js

```diff
@@ -219,18 +207,17 @@ export const handleTooltipClose = () => {
  * @todo - describe object's function
  */
 export const validationSchema = Yup.object().shape({
-  username: Yup.string().required('required'),
-  user_location: Yup.string().min(1, 'min').required('required'),
-  password: Yup.string().required('required'),
-  email: Yup.string().email('invalid').when('phone', {
-    is: (phone) => !phone || phone.length === 0,
-    then: Yup.string().required('phoneOrEmail')
-  }),
-  phone: Yup.string().when('email', {
-    is: (email) => !email || email.length === 0,
-    then: Yup.string().required('phoneOrEmail')
-  }).test('phone_is_invalid', 'invalid', function (value) {
-    return /^[+][0-9]{9,15}$/g.test(value) || !value ? true : false;
-  }),
-  bio: Yup.string().max(255, 'tooLong'),
-}, ['phone', 'email']);
\ No newline at end of file
+  username: Yup.string().required('Username is required'),
+  user_location: Yup.string().required('Location is required'),
+  password: Yup.string().required('Password is required'),
+  email: Yup.string().email('Invalid email'),
+  phone: Yup.string()
+    .when('email', {
+      is: email => !email || email.length === 0,
+      then: () => Yup.string().required('Provide an email or phone'),
```

Comment:
I think all these validations need to be translated.

Reply:
I think we can ignore translations for now

Reply:
no we can't

Reply:
Alright, let me revert everything. But really the error messages before where not informative, we had something like required, am sure you've seen the change log. 
What  I know is that we have alot of untranslated strings for sure. Translations can come up as a single PR

Reply:
those are translation keys @coderatomy . Things like `required` and `PhoneOrEmail` are not just strings but translation keys that can be found in the translation files. If it's failing to translate then something else is likely wrong and we'd have to debug and fix this issue.